R/input-radiobuttons.R

```diff
@@ -108,6 +108,11 @@ radioButtons <- function(inputId, label, choices = NULL, selected = NULL,
     style = if (!is.null(width)) paste0("width: ", validateCssUnit(width), ";"),
     class = divClass,
     shinyInputLabel(inputId, label),
+    # Improve accessibility (#3105)
+    # https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/radio/radio-1/radio-1.html
+    role = "radiogroup",
+    # The `aria-labelledby` is tied to id value in `shinyInputLabel()` to be announced to screen readers and in Braille:
+    `aria-labelledby` = paste0(inputId, "-label"),
```

Comment:
Probably safer to extract this value from the object returned by `shinyInputLabel(inputId, label)`